src/Domain/Creators/TimeDependentOptions/TranslationMap.hpp

```diff
@@ -22,29 +22,42 @@ namespace domain::creators::time_dependent_options {
  * coefficients.
  */
 template <size_t Dim>
-struct TranslationMapOptions {
-  using type = Options::Auto<TranslationMapOptions, Options::AutoLabel::None>;
-  static std::string name() { return "TranslationMap"; }
+struct TranslationMapHardcodedOptions {
```

Comment:
I always read "hardcore options" 💪

Suggestion: keep this class named `TranslationMapOptions`. And the tag below name just `TranslationMap` because it's already in the `time_dependent_options` namespace.